src/test/java/nextstep/subway/acceptance/MemberAcceptanceTest.java

```diff
@@ -67,10 +86,28 @@ void deleteMember() {
     @DisplayName("회원 정보를 관리한다.")
     @Test
     void manageMember() {
+        accessToken = 로그인_되어_있음(ADMIN_EMAIL, ADMIN_PASSWORD);
+        ExtractableResponse<Response> 조회응답 = 회원_정보_조회_요청(회원생성_Response);
+        회원_정보_조회됨(조회응답, EMAIL, AGE);
+
+        ExtractableResponse<Response> 수정응답 = 회원_정보_수정_요청(accessToken, 회원생성_Response, EMAIL+"_edit", PASSWORD+"_edit", AGE);
+        assertThat(수정응답.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        회원_정보_조회_요청(회원생성_Response);
+
+        ExtractableResponse<Response> 삭제응답 = 회원_삭제_요청(accessToken, 회원생성_Response);
```

Comment:
가독성을 좋게 하기 위해서 아래와 같이 해주는 방법도 있습니다 :) 
```suggestion
        var 삭제응답 = 회원_삭제_요청(accessToken, 회원생성_Response);
```